services/navigation/builtin/builtin.go

```diff
@@ -553,6 +554,10 @@ func (svc *builtIn) startWaypointMode(ctx context.Context, extra map[string]inte
 					svc.logger.Infof("skipping waypoint %+v since it was deleted", wp)
 					continue
 				}
+				if errors.Is(err, errRetry) {
```

Comment:
I dont understand this, it doesnt look like anything would set this error.  Moreover, behavior from nav should be retrying any motion failure, no?

Reply:
seconded; this isn't a thing that we produce right now and should be removed.

IIRC we have a ticket open to retry the waypoint if we error rather than skipping, but this doesn't do that.